systemtest/src/main/java/io/strimzi/systemtest/utils/kafkaUtils/KafkaUtils.java

```diff
@@ -199,20 +199,25 @@ public static void waitForClusterStability(String namespaceName, String clusterN
             boolean kafkaSameAsLast = kafkaSnaptop.equals(kafkaPods[0]);
             boolean eoSameAsLast = eoSnapshot.equals(eoPods[0]);
             if (!zkSameAsLast) {
-                LOGGER.info("ZK Cluster not stable");
+                LOGGER.warn("ZK Cluster not stable");
             }
             if (!kafkaSameAsLast) {
-                LOGGER.info("Kafka Cluster not stable");
+                LOGGER.warn("Kafka Cluster not stable");
             }
             if (!eoSameAsLast) {
-                LOGGER.info("EO not stable");
+                LOGGER.warn("EO not stable");
             }
             if (zkSameAsLast
                     && kafkaSameAsLast
                     && eoSameAsLast) {
                 int c = count[0]++;
-                LOGGER.info("All stable for {} polls", c);
-                return c > 60;
+                LOGGER.debug("All stable after {} polls", c);
+                if (c > 60) {
+                    LOGGER.info("Kafka cluster is stable after {} polls.", c);
+                    return true;
+                } else {
+                    return false;
+                }
```

Comment:
You can simplify it like this:
```suggestion
                if (c > 60) {
                    LOGGER.info("Kafka cluster is stable after {} polls.", c);
                    return true;
                }
                return false;
```

Reply:
fixed. thanks